README.md

```diff
@@ -0,0 +1,48 @@
+# Warehouse Logistics Service
+
+Hermes WLS (Warehouse and Logistics Service) functions as a middleware between NLNs (National Library of Norway) catalogues and storage systems.
+The goal with the service is to unite the different storage systems and magazines under the same service.
+
+
+## Running the application
+
+Hermes WLS runs on Java 21.
```

Comment:
Too much information, need to condense it even more 😉

Reply:
I could remove the "21"